.github/workflows/ci.yml

```diff
@@ -21,6 +21,7 @@ jobs:
         strategy:
             matrix:
                 php-version:
+                    - '8.0.16'
                     - '8.0.14'
```

Comment:
No 8.0.15 ?

Reply:
8.0.15 does not compile with the socket extension, but this got fixed with 8.0.16